File: eCarService.Service/Database/Rating.cs
using System;
using System.Collections.Generic;

namespace eCarService.Database
{
    public partial class Rating
    {
        public int RatingId { get; set; }
        public int Rate { get; set; }
        public string? Comment { get; set; }
        public int? OfferId { get; set; }
        public int? UserId { get; set; }

        public virtual Offer? Offer { get; set; }
        public virtual User? User { get; set; }
    }
}

File: ecarservice_mobile/lib/model/rating.dart
import 'package:json_annotation/json_annotation.dart';
part 'rating.g.dart';

@JsonSerializable()
class Rating {
  int? ratingId;
  int? rate;
  String? comment;
  int? offerId;
  int? userId;

  Rating() {}

  factory Rating.fromJson(Map<String, dynamic> json) => _$RatingFromJson(json);

  /// Connect the generated [_$RatingToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$RatingToJson(this);
}
